test/views/behaviours/ViewBehaviours.scala

```diff
@@ -31,22 +34,15 @@ trait ViewBehaviours extends ViewSpecBase {
       "rendered" must {
 
         "have the correct banner title" in {
-
-          val doc = asDocument(view)
-          val bannerTitle = doc.getElementsByClass("hmrc-header__service-name hmrc-header__service-name--linked")
-          bannerTitle.html() mustBe messages("service.name")
+          findBannerTitle(view) mustBe messages("service.name")
         }
 
         "display the correct browser title" in {
-
-          val doc = asDocument(view)
-          assertEqualsMessage(doc, "title", sectionKey, s"$messageKeyPrefix.title")
+          findBannerTitle(view) mustBe messages("service.name")
         }
 
         "display the correct page title" in {
-
-          val doc = asDocument(view)
-          assertPageTitleEqualsMessage(doc, s"$messageKeyPrefix.heading")
+          findBannerTitle(view) mustBe messages("service.name")
```

Comment:
Again, this looks like it's now checking the banner title against the `service.name` message but the original test was checking for `.heading`.